src/omnisharp/launcher.ts

```diff
@@ -408,7 +408,7 @@ function launchNix(launchPath: string, cwd: string, args: string[]): LaunchResul
 
 function launchNixMono(launchPath: string, cwd: string, args: string[], environment: NodeJS.ProcessEnv, useDebugger: boolean): LaunchResult {
     let argsCopy = args.slice(0); // create copy of details args
-    argsCopy.unshift(launchPath);
+    argsCopy.unshift(`$"{launchPath}"`);
```

Comment:
Isn't it a typo?

```suggestion
    argsCopy.unshift(`"${launchPath}"`);
```

Reply:
@333fred ^

Reply:
@vchirikov no need to ping me so many times, I'm watching this issue :).

Yes, that does indeed look suspicious. However, this is launching mono on Linux or Unix, not anything to do with Windows.